### PR TITLE
chore(all): remove vestiges of v1 support

### PR DESCRIPTION
**Pull request by biffgaut**
Description of changes:
There are still scripts left from when we supported V1 and V2 simultaneously, and some scripts made more complex for that support. V1 is long since deprecated, we can remove this support and simplify our scripts.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: githubautobuild-for-cdk-v2
* Commit ID: bd9c717d767de80102965383438306ac05b8c162
* Result: SUCCEEDED
* [Build Logs](https://t2ulkoluaf.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=6dfc966e-f728-44da-b3b4-8aeae4174fb6%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: githubautobuild-for-cdk-v2
* Commit ID: acbd70b06f046cf74b0ccb34f29e0944f64d850c
* Result: SUCCEEDED
* [Build Logs](https://t2ulkoluaf.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=03193551-75a8-4a3f-b829-160a6fe11c96%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: source/package.json [hunk 1]
```diff
@@ -27,10 +27,13 @@
     "jsii-pacmak": "^1.92.0",
     "tslint": "^5.20.1",
     "typescript": "4.7.4",
-    "aws-cdk-migration": "^1.135.0"
+    "aws-cdk-migration": "^1.135.0",
+    "constructs": "10.0.0"
```
Comment (georgebearden) [inline source/package.json:31]: pinning constructs to `10.0.0` explicitly, without allowing for minor or patch upgrades seems intentional, so just curious about the reasoning?

**Review by georgebearden — APPROVED** [review]
looks good to me.

**Pull request merged**